src/Percy.js

```diff
@@ -27,6 +28,18 @@ class Percy {
     this.environment = new PercyEnvironment(process.env);
     this.loaders = loaders;
     this.includeBase = includeBase;
+
+    if (process.env.PERCY_ENABLE === '0') {
```

Comment:
I think this should be just `==` so we don't run into any string/integer mismatch issues.

It's just `==` in `ember-percy` 
https://github.com/percy/ember-percy/blob/5bf1bbebb69d61cfc06e16a607a76b2c443572f4/index.js#L122